firmware/application/apps/ui_recon.cpp

```diff
@@ -29,6 +29,29 @@ using portapack::memory::map::backup_ram;
 
 namespace ui {
 
+void ReconView::colorize_waits() {
+    // colorize wait on match
+    if (wait == 0) {
+        field_wait.set_style(&style_blue);
+    } else if (wait >= 500) {
+        field_wait.set_style(&style_white);
+    } else if (wait > -500 && wait < 500) {
+        field_wait.set_style(&style_red);
+    } else if (wait <= -500) {
+        field_wait.set_style(&style_green);
+    }
+    // colorize lock time if in SPARSE mode as in continuous the lock_wait time is disarmed at first lock count
+    if (recon_match_mode == RECON_MATCH_SPARSE) {
```

Comment:
nit: consider tidying up

```
auto style = &white;

if (... your condition ...)
    style = &yellow;

field_lock_wait.set_style(style);
```

or

```
bool use_yellow = (your condition test);
field_lock_wait.set_style(use_yellow ? &yellow : &white);
```

Reply:
I'm ok with the suggestion, but sometimes I think it's better to see clearly what's happening.
Thanks for your comments/ review :-)